.github/workflows/uptodate.yaml

```diff
@@ -0,0 +1,32 @@
+---
+name: Is up to date
```

Comment:
Desserve a description :

Compare branch prod-2-7-advance to prod-2-7 except some files removed in file ci/cleanup.